.github/workflows/_test.yml

```diff
@@ -38,12 +38,16 @@ jobs:
 
       - name: Install dependencies
         shell: bash
-        run: poetry install
+        run: poetry install --with=test_integration
```

Comment:
we should install these extras after the core tests finish. I think there's a few unit tests to confirm that importing without the extra deps installed doesn't crash (only using objects does) - e.g. test_dependencies.py

Reply:
great pt